src/fs.ts

```diff
@@ -48,21 +48,17 @@ export async function writeJson(
   data: object,
   cwd = '.',
 ): Promise<void> {
-  try {
-    const text = JSON.stringify(data, null, '  ');
-    return writeText(path, text, cwd);
-  } catch (e) {
-    return;
-  }
+  const text = JSON.stringify(data, null, '  ');
+  return await writeText(path, text, cwd);
 }
 
 type ListOptions = GlobOptions & {
   extensions?: string[];
 };
 
 export async function list(
-  pattern = '**/*',
-  cwd = '.',
+  pattern: string,
+  cwd: string,
```

Comment:
`list` is never called with the defaults and using them made writing tests for it default as the default `cwd` meant it would list all of the files in this project, so we opted to remove the default, negating the issue.